Source/Shaders/ModelExperimental/GeometryStageVS.glsl

```diff
@@ -2,8 +2,20 @@ void geometryStage(inout ProcessedAttributes attributes)
 {
     // Compute positions in different coordinate systems
     vec3 positionMC = attributes.positionMC;
+
+    mat3 normal;
+    mat4 modelView;
+
+    #ifdef USE_LEGACY_INSTANCING
```

Comment:
I could potentially pull this code back out in `ModelExperimentalVS.glsl`.

Something like:

```glsl
void main() 
{
// ...
   mat3 normal = czm_normal;
   mat4 modelView = czm_modelView;

    // Update the position for this instance in place
    #ifdef HAS_INSTANCING
    
        #ifdef USE_LEGACY_INSTANCING
        normal = instanceModelViewInverseTranspose;
        modelView = instanceModelView;
        legacyInstancingStage(attributes.positionMC);
        #else
        instancingStage(attributes.positionMC);
        #endif

        #ifdef USE_PICKING
        v_pickColor = a_pickColor;
        #endif

    #endif

    // Compute the final position in each coordinate system needed.
    // This also sets gl_Position.
    geometryStage(attributes, normal, modelView);    
// ...

```

Reply:
Yes, the example here is wrong. In the update code, I chose to use extra variable to pass into `legacyInstancingStage`, instead of passing the `normal` and `modelView` directly, because I think it makes the code more readable.